vcd/resource_vcd_org_user.go

```diff
@@ -223,64 +223,21 @@ func resourceToOrgUser(d *schema.ResourceData, meta interface{}) (*govcd.OrgUser
 // Fills a ResourceData container with data retrieved from an OrgUser and an AdminOrg
 // Used after retrieving the user (read, import), to fill the Terraform container appropriately
 func setOrgUserData(d *schema.ResourceData, orgUser *govcd.OrgUser, adminOrg *govcd.AdminOrg) error {
-
 	d.SetId(orgUser.User.ID)
-	err := d.Set("org", adminOrg.AdminOrg.Name)
-	if err != nil {
-		return err
-	}
-	err = d.Set("name", orgUser.User.Name)
-	if err != nil {
-		return err
-	}
-	err = d.Set("provider_type", orgUser.User.ProviderType)
-	if err != nil {
-		return err
-	}
-	err = d.Set("is_group_role", orgUser.User.IsGroupRole)
-	if err != nil {
-		return err
-	}
-	err = d.Set("description", orgUser.User.Description)
-	if err != nil {
-		return err
-	}
-	err = d.Set("full_name", orgUser.User.FullName)
-	if err != nil {
-		return err
-	}
-	err = d.Set("email_address", orgUser.User.EmailAddress)
-	if err != nil {
-		return err
-	}
-	err = d.Set("telephone", orgUser.User.Telephone)
-	if err != nil {
-		return err
-	}
-	err = d.Set("instant_messaging", orgUser.User.IM)
-	if err != nil {
-		return err
-	}
-	err = d.Set("enabled", orgUser.User.IsEnabled)
-	if err != nil {
-		return err
-	}
-	err = d.Set("is_locked", orgUser.User.IsLocked)
-	if err != nil {
-		return err
-	}
-	err = d.Set("deployed_vm_quota", orgUser.User.DeployedVmQuota)
-	if err != nil {
-		return err
-	}
-	err = d.Set("stored_vm_quota", orgUser.User.StoredVmQuota)
-	if err != nil {
-		return err
-	}
-	err = d.Set("role", orgUser.User.Role.Name)
-	if err != nil {
-		return err
-	}
+	_ = d.Set("org", adminOrg.AdminOrg.Name)
```

Comment:
Please elaborate why the old error checking is not needed anymore.

Reply:
We had a team discussion and we agreed to skip the error checking, at least in places where we are assigning a simple value. After that decision, though, I found a few places where assigning a complex structure would lead to errors. For scalar assignments it should not be a problem if we have tests that prove the correctness of the values, which in this case we do.